src/SprykerEco/Client/OpenAi/Chat/OpenAiChat.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Client\OpenAi\Chat;
+
+use Generated\Shared\Transfer\OpenAiChatRequestTransfer;
+use Generated\Shared\Transfer\OpenAiChatResponseTransfer;
+use OpenAI\Client;
+use OpenAI\Responses\Chat\CreateResponse;
+use SprykerEco\Client\OpenAi\OpenAiConfig;
+
+class OpenAiChat implements OpenAiChatInterface
+{
+    /**
+     * @var string
+     */
+    protected const OPENAI_MESSAGE_ROLE_KEY = 'role';
+
+    /**
+     * @var string
+     */
+    protected const OPENAI_MESSAGE_MODEL_KEY = 'model';
+
+    /**
+     * @var string
+     */
+    protected const OPENAI_MESSAGE_MESSAGES_KEY = 'messages';
+
+    /**
+     * @var string
+     */
+    protected const OPENAI_MESSAGE_CONTENT_KEY = 'content';
+
+    /**
+     * @var string
+     */
+    protected const OPENAI_MESSAGE_ROLE_USER_VALUE = 'user';
+
+    /**
+     * @var \OpenAI\Client
+     */
+    protected Client $openAiClient;
```

Comment:
Should be a bridge

Reply:
I'm not sure about it, couldn't find a single bridge example for external dependency. I.e `Spryker\Service\FlysystemAws3v3FileSystem\Model\Builder\Adapter\Aws3v3AdapterBuilder` instantiates S3 Client directly inside the class. I would add an abstraction layer - something like `OpenAiClientProviderPlugin` and then connect it throught DependencyProvider, or straight up instantiate OpenAi `Client` directly inside the dependency provider. But bridge does not seems suitable here 🤔